pkg/kubelet/status/manager.go

```diff
@@ -217,6 +212,31 @@ func (m *manager) SetContainerReadiness(podUID types.UID, containerID kubecontai
 	m.updateStatusInternal(pod, status, false)
 }
 
+func findContainerStatus(status *api.PodStatus, containerID string) (containerStatus *api.ContainerStatus, init bool, ok bool) {
+	// Find the container to update.
+	containerIndex := -1
+	for i, c := range status.ContainerStatuses {
```

Comment:
InitContainer ~= InitContainerStatuses as Container =~ ContainerStatuses.  The parallel works more cleanly I think.